src/gpuocean/SWEsimulators/CDKLM16pair.py

```diff
@@ -0,0 +1,240 @@
+# -*- coding: utf-8 -*-
+
+"""
+This software is part of GPU Ocean. 
+
+Copyright (C) 2016 SINTEF ICT, 
+Copyright (C) 2017-2019 SINTEF Digital
+Copyright (C) 2017-2019 Norwegian Meteorological Institute
+
+This python module implements the finite-volume scheme proposed by
+Alina Chertock, Michael Dudzinski, A. Kurganov & Maria Lukacova-Medvidova (2016)
+Well-Balanced Schemes for the Shallow Water Equations with Coriolis Forces
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+#Import packages we need
+import numpy as np
+from scipy import interpolate 
+
+# Needed for the random perturbation:
+import pycuda.driver as cuda
+from gpuocean.SWEsimulators import Simulator, OceanStateNoise
+from gpuocean.utils import Common, NetCDFInitialization, OceanographicUtilities
+
+
+class CDKLM16pair():
+    """
```

Comment:
Is this file related to multi-res, and not multi-level? Should it then be part of this PR? @FlorianBeiser